app/src/main/resources/common/README.md.mustache

```diff
@@ -121,12 +137,36 @@ Or via:
 java -Xdebug -Xrunjdwp:transport=dt_socket,address=5000,server=y,suspend=y -jar build/libs/{{appName}}.war
 ```
 
-Run the CAS web application as a *standalone* executable WAR:
+## Build and run CAS as a *standalone* executable WAR
+
+This allows you to run the CAS web application as a *standalone* executable WAR on Linux and Mac.
+
+The build system will add a startup script inside the war, that makes it executable.
+
+### Building
+
+In order to build a standalone executable war, update `gradle.properties` and set `executable=true`, then run:
 
 ```bash
-./gradlew[.bat] clean executable
+./gradlew clean executable
 ```
 
+### Running
+
+Once built, the generated `build/libs/cas.war` will be directly executable on Linux and Mac:
+
+```bash
+./build/libs/cas.war
+```
+
+You can take a look at the startup script inside the war by running the command:
```

Comment:
This bit is also not maintainable. This behavior could change at any moment by the external library that handles this sort of thing, and we can manage the change automatically, or per CAS version in the docs like this. Explanations like this belong in a blog.